src/routes/index.ts

```diff
@@ -30,15 +30,16 @@ import {
   signOut
 } from "../controllers";
 
-import { serviceAvailabilityMiddleware } from "../middleware/service.availability.middleware";
+// import { serviceAvailabilityMiddleware } from "../middleware/service.availability.middleware";
```

Comment:
So yes, as discussed this needs to remain now it's been clarified.

Reply:
> 

Thanks @mwestacott, I will fix it first think in the morning. Do you know how to dismiss your approval?

Reply:
Done